docs/provisioning/README.md

```diff
@@ -0,0 +1,52 @@
+# AzureML FL provisioning cookbook
+
+This page provides a "pick and choose" guide to provisioning a new FL environment. Each section provides multiple alternatives depending on your setup. Some sections are optional and can be skipped if you already have the required infrastructure in place.
+
+A lot of those steps are still marked :construction:, we are actively working to provide them im the coming iterations.
+
+## Prerequisites
+
+To enjoy these quickstart, you will need to:
+- have an active [Azure subscription](https://azure.microsoft.com) that you can use for development purposes,
+- have permissions to create resources, set permissions, and create identities in this subscription (or at least in one resource group),
+  - Note that to set permissions, you typically need _Owner_ role in the subscription or resource group - _Contributor_ role is not enough. This is key for being able to _secure_ the setup.
+- [install the Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli).
+
+
+## Create an Azure ML Workspace
+
+Creating an Azure ML workspace is the starting point to create your full Federated Learning environment. Below are two options you could use, drawing from the existing documentation.
```

Comment:
Maybe add a sentence explaining what it will be used for? Can be super short, like "This is where the orchestrator will live, and where the FL runs can be managed and analyzed.".

Reply:
very good point, I'll add something.